app/app/src/main/java/com/globant/calculator/android/mvp/presenter/CalculatorPresenter.java

```diff
@@ -13,4 +21,53 @@ public CalculatorPresenter(CalculatorModel model, CalculatorView view) {
         this.view = view;
     }
 
+    public void onClearButtonPress() {
+        model.clear();
+        view.showOperationPressed(NUMBER_ZERO);
+        view.showResult(ZERO_INT);
+    }
+
+    public void onAddButtonPressed(String symbol) {
```

Comment:
unify all operators in only one method.. just like the onNumberButtonPressed()

Reply:
done